common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -2284,54 +2292,51 @@ func (d *cassandraPersistence) LeaseTaskQueue(request *p.LeaseTaskQueueRequest)
 func (d *cassandraPersistence) UpdateTaskQueue(request *p.UpdateTaskQueueRequest) (*p.UpdateTaskQueueResponse, error) {
 	tli := *request.TaskQueueInfo
 	tli.LastUpdateTime = timestamp.TimeNowPtrUtc()
-	if tli.Kind == enumspb.TASK_QUEUE_KIND_STICKY { // if task_queue is sticky, then update with TTL
-		expiry := types.TimestampNow()
```

Comment:
Before your change, was this even used anywhere?

Reply:
nop, ref: 
https://github.com/uber/cadence/blob/0.11.x/common/persistence/cassandra/cassandraPersistence.go#L2384